index.rst

```diff
@@ -196,13 +196,22 @@ Management and Monitoring
 
     Debug, components/debug, bug-report.svg, dark-invert
     Logger, components/logger, file-document-box.svg, dark-invert
-    OTA Updates, components/ota, system-update.svg, dark-invert
-    OTA Updates via HTTP Request, components/ota_http_request, system-update.svg, dark-invert
     Prometheus, components/prometheus, prometheus.svg
     Safe Mode, components/safe_mode, restart-alert.svg
     Web Server, components/web_server, http.svg, dark-invert
     ESP32 Camera Web Server, components/esp32_camera_web_server, camera.svg, dark-invert
 
+Update Installation
+-------------------
+
+Install updates over-the-air (OTA).
+
+.. imgtable::
+
+    OTA Core, components/ota, system-update.svg, dark-invert
+    OTA Updates, components/ota_esphome, system-update.svg, dark-invert
+    OTA Updates via HTTP Request, components/ota_http_request, system-update.svg, dark-invert
```

Comment:
```suggestion
    OTA Core, components/ota/index, system-update.svg, dark-invert
    OTA Updates, components/ota/esphome, system-update.svg, dark-invert
    OTA Updates via HTTP Request, components/ota/http_request, system-update.svg, dark-invert
```

Reply:
Done in [`3324ee7` (#3976)](https://github.com/esphome/esphome-docs/pull/3976/commits/3324ee72fbce7c6aa7a1b859d8ae92760134e7ae)

Reply:
...and [`630784e` (#3976)](https://github.com/esphome/esphome-docs/pull/3976/commits/630784edddbf31d7c66d24afdaf9a21031c2e069)